br/pkg/lightning/mydump/parquet_parser.go

```diff
@@ -26,6 +27,8 @@ const (
 	// if a parquet if small than this threshold, parquet will load the whole file in a byte slice to
 	// optimize the read performance
 	smallParquetFileThreshold = 256 * 1024 * 1024
+	jan011970                 = 2440588
```

Comment:
what's the meaning of 2440588

Reply:
The decoding of the INT96 timestamp refers to a parquet library called [parquet-go](https://github.com/fraugster/parquet-go). Will paste some comments here tomorrow.

Reply:
2536ce5